docs/content/dagster-cloud/managing-deployments/managing-multiple-projects-and-teams.mdx

```diff
@@ -0,0 +1,177 @@
+
+#### Example with GitHub CI/CD on Hybrid deployment
+
+1. For each repository, use the CI/CD workflow provided in [Dagster Cloud Hybrid Quickstart](https://github.com/dagster-io/dagster-cloud-hybrid-quickstart/blob/main/.github/workflows/dagster-cloud-deploy.yml).
+
+2. For each project in the repository, you can configure the [code location to deploy in the `dagster_cloud.yml` file](/dagster-cloud/managing-deployments/dagster-cloud-yaml) and configure it accordingly to your project structure.
+
+   ```yaml
+   # File: dagster_cloud.yml
+   locations:
+     - location_name: project_a
+       code_source:
+         package_name: project_a
+       build:
+         # ...
+     - location_name: project_b
+       code_source:
+         package_name: project_b
+       build:
+         # ...
+   ```
+
+3. Modify the CI/CD workflow to deploy every code locations for this repository.
+
+   ```yaml
+   # File: .github/workflows/dagster-cloud-deploy.yml
+   # ...
+   jobs:
+     dagster-cloud-deploy:
+       # ...
+       steps:
+         - name: Update build session with image tag for "project_a" code location
+           id: ci-set-build-output-project-a
+           if: steps.prerun.outputs.result != 'skip'
+           uses: dagster-io/dagster-cloud-action/actions/utils/dagster-cloud-cli@v0.1
+           with:
+             command: "ci set-build-output --location-name=project_a --image-tag=$IMAGE_TAG"
+
+         - name: Update build session with image tag for "project_b" code location
+           id: ci-set-build-output-project-b
+           if: steps.prerun.outputs.result != 'skip'
+           uses: dagster-io/dagster-cloud-action/actions/utils/dagster-cloud-cli@v0.1
+           with:
+             command: "ci set-build-output --location-name=project_b --image-tag=$IMAGE_TAG"
+         # ...
+   ```
+
+## Strategies for execution context isolation
+
+Separating execution context between projects can have several motivations:
+
+- facilitate separation of duty between teams to prevent access to sensitive data
+- different compute environments and requirements (different architecture, cloud provider, etc.)
+- prevent noisy neighbors effects (a project with a large number of runs can impact the performance of other projects)
+
+There are three isolation levels you may want to consider:
+
+### Code location isolation
+
+<Image
+alt="Diagram of isolation at the code location level"
+src="/images/dagster-cloud/managing-deployments/isolation-level-code-locations.png"
+width={1167}
+height={384}
+/>
+
+If you have no specific requirements for isolation beyond the ability to deploy and run multiple projects, you can use a single agent and deployment to manage all your projects as individual code locations.
+
+Pro:
+
+- simplest and most cost-effective solution
+- user access control can be set at the code location level
+- single glass pane to view all assets
+
+Con:
+
+- no isolation between execution environments
```

Comment:
@slopp  pointed out that it might be worth including what you need to do here to reference assets between the multiple code locations (using source assets)